.github/workflows/deploy-to-azure-app-service.yml

```diff
@@ -126,7 +126,13 @@ jobs:
         - ${{ inputs.timeout-minutes }}
     timeout-minutes: ${{ matrix.timeout-minutes }}
     steps:
-    - uses: actions/checkout@v3
+    - name: Setup Scripts
+      shell: pwsh
+      run: |
+        "${{ github.action_path }}" >> $Env:GITHUB_PATH
+        (Resolve-Path "${{ github.action_path }}/../../../Scripts").Path >> $Env:GITHUB_PATH
```

Comment:
`action_path` won't work in a workflow (as opposed to an action). But you needn't refactor the `set-up-runtime-option` to an action for this; instead, the step's output can be passed around in an environment variable.